src/secureCreatives.js

```diff
@@ -69,39 +67,13 @@ export function receiveMessage(ev) {
   }
 }
 
-function handleRenderRequest(reply, data, adObject) {
-  if (adObject == null) {
-    emitAdRenderFail({
-      reason: constants.AD_RENDER_FAILED_REASON.CANNOT_FIND_AD,
-      message: `Cannot find ad for cross-origin render request: '${data.adId}'`,
-      id: data.adId
-    });
-    return;
-  }
-  if (adObject.status === constants.BID_STATUS.RENDERED) {
-    logWarn(`Ad id ${adObject.adId} has been rendered before`);
-    events.emit(STALE_RENDER, adObject);
-    if (deepAccess(config.getConfig('auctionOptions'), 'suppressStaleRender')) {
-      return;
-    }
-  }
-
-  try {
-    _sendAdToCreative(adObject, reply);
-  } catch (e) {
-    emitAdRenderFail({
-      reason: constants.AD_RENDER_FAILED_REASON.EXCEPTION,
-      message: e.message,
-      id: data.adId,
-      bid: adObject
-    });
-    return;
-  }
-
-  // save winning bids
-  auctionManager.addWinningBid(adObject);
-
-  events.emit(BID_WON, adObject);
+function handleRenderRequest(reply, message, bidResponse) {
+  handleRender(function (adData) {
+    resizeRemoteCreative(bidResponse);
```

Comment:
Would this resize for outstream renders? My assumption here is that it could if the bid has video dimensions defined as the width/height and could cause white space, but perhaps I am overlooking something

Reply:
video doesn't go through this, both `pbjs.renderAd` and PUC throw an error if they're asked to render a video bid - since they don't know the player to use or how to interact with it. The flow for it is very different and I think it wouldn't result in a GPT frame unless you set up the player in your creative. 
Our guides for it do not use GPT at all and instead generate a VAST url that the pub is expected to submit to their player (or the new-ish video module can do it for them, but still without a separate creative iframe that could send messages here).

Reply:
I think I misunderstood your question - if an outstream renderer is in use and the frame here is therefore useless, are we resizing it to be large and empty? I will do some testing and report back.

Reply:
The answer is no, when bids have a renderer this code path is skipped - I added a test case for it.

Reply:
Ah yep I see. Thanks for confirming!